filebeat/docs/docker-input.asciidoc

```diff
@@ -0,0 +1,64 @@
+:type: docker
+
+[[docker-input]]
+=== Docker input
+
+++++
+<titleabbrev>Docker</titleabbrev>
+++++
+
+experimental[]
+
+Use the `docker` input to read logs from Docker containers. 
+
+This input searches for container logs under its path, and parse them into
+common message lines, extracting timestamps too. Everything happens before line
+filtering, multiline, and JSON decoding, so this input can be used in
+combination with those settings.
+
+Example configuration:
+
+// REVIEWERS: The filebeat.reference.yml file does not show a docker input type
+// so I'm not sure if the following config is correct. docker only shows up under
+// autodiscover.
```

Comment:
@exekias can you have a look?

Reply:
The example configuration is correct :+1: